wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/elevatorsimulation/Robot.java

```diff
@@ -29,11 +31,20 @@ public class Robot extends TimedRobot {
   private static final int kEncoderBChannel = 1;
   private static final int kJoystickPort = 0;
 
-  private static final double kElevatorKp = 5.0;
+  private static final double kElevatorKp = 5;
+  private static final double kElevatorKi = 0;
+  private static final double kElevatorKd = 0;
+
+  private static final double kElevatorkS = 0.0; // volts
+  private static final double kElevatorkG = 0.762; // volts
+  private static final double kElevatorkV = 0.762; // volt seconds per meters
+  private static final double kElevatorkA = 0.0; // volt seconds² per meters
```

Comment:
The more intuitive way to describe these is as volts per velocity and volts per acceleration. Therefore, the units should be V/(m/s) and V/(m/s²) respectively.